src/store/index.ts

```diff
@@ -7,9 +7,9 @@ import { EmbalmActions } from './embalm/actions';
 import { embalmInitialState, embalmReducer, EmbalmState } from './embalm/reducer';
 import { SarcophagusActions } from './sarcophagus/actions';
 import {
-  sarcophagusInitialState,
-  sarcophagusReducer,
-  SarcophagusState,
+  sarcophagiInitialState,
+  sarcophagiReducer,
+  SarcophagiState,
```

Comment:
I renamed "Sarcophagus" to "Sarcophagi" with respect to store/reducer/state, as that had started to become confusing when dealing with the `SarcophagusState` type. Also more consistent with other parts of the codebase working with sarcophagi in general